samples/connectors/src/main/java/quarks/samples/connectors/iotf/IotfSensors.java

```diff
@@ -109,6 +99,31 @@ public static void simulatedSensors(IotDevice device, boolean print) {
         // with event identifier "sensors".
         device.events(sensors, "sensors", QoS.FIRE_AND_FORGET);
     }
+    
+    /**
+     * Create a heart beat device event with
+     * identifier {@code heartbeat} to
+     * ensure there is some immediate output and
+     * the connection to IoTF happens as soon as possible.
+     * @param device IoT device
+     */
+    public static void heartBeat(IotDevice device, boolean print) {
```

Comment:
Any thoughts on how this sample method comment could be improved to not promote its use? It does already state what the purpose of the heartbeat is for this sample.

Not also that a heartbeat device event performs a somewhat different role to KeepAlive/LastWillAndTestament in that subscribers have visibility to the "I'm still connected" events